q2_vsearch/_format.py

```diff
@@ -0,0 +1,21 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import qiime2.plugin.model as model
+
+
+class UchimeStatsFmt(model.TextFileFormat):
+    def sniff(self):
```

Comment:
This is technically deprecated, but it's not worth worrying about right now.

Reply:
Here's a [good example](https://github.com/qiime2/q2-types/blob/0f20f1c8d77d50a1d8335fa524b99f75928ab536/q2_types/per_sample_sequences/_format.py#L179-L197) if you'd like to switch it over.

Reply:
And also [here](https://github.com/qiime2/q2-types/blob/0f20f1c8d77d50a1d8335fa524b99f75928ab536/q2_types/per_sample_sequences/_format.py#L153)